### PR TITLE
Track w3swap on pego-network and add new protocol w3bank

### PR DESCRIPTION
**Pull request by bhowedev**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/w3swap: 
        
 
  
    
    
    Error: 
        Unknown chain(s): pego
        Note: if you think that the chain is correct but missing from our list, please add it to 'projects/helper/chains.json' file

File: projects/helper/chains.json [hunk 1]
```diff
@@ -153,6 +153,7 @@
   "palm",
   "parallel",
   "persistence",
+  "pg",
```
Comment (Define101) [inline projects/helper/chains.json:156]: can you change it to "pego"

File: projects/w3bank/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+  markets.forEach(market => {
+    api.add(market['underlying'], market['borrowed']);
+  });
+}
+
+module.exports = {
+  timetravel: true,
+  start: 1693843200,
+  pg: {
```
Comment (Define101) [inline projects/w3bank/index.js:86]: "pego"

**Comment by llamatester** [comment]
The adapter at projects/w3swap exports TVL: 
        
 
 
    pg                        31.72 M
    bsc                       254.75 k
    
    total                    31.97 M

**Comment by llamatester** [comment]
The adapter at projects/w3bank exports TVL: 
        
 
 
    pg                        0
    pg-borrowed               0
    borrowed                  0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/w3swap exports TVL: 
        
 
 
    bsc                       254.53 k
    pg                        31.53 M
    
    total                    31.79 M

**Comment by llamatester** [comment]
The adapter at projects/w3bank exports TVL: 
        
 
 
    pg                        1.50 M
    pg-borrowed               0
    borrowed                  0
    
    total                    1.50 M

**Comment by g1nt0ki** [comment]
@bhowedev I have not added token mapping for USDT, is there some way to verify they USDT brdiged to pego network, does it all come from bsc, if so maybe you can share the bridge contract?

**Pull request merged**